eth_interface.md

```diff
@@ -377,7 +377,9 @@ account with non-empty associated code.
 
 ## return
 
-Halt execution returning output data.
+Set the returning output data for the execution.
+
+*Note*: multiple invocations will overwrite the previous data.
```

Comment:
`throw Error('return')` I assumed that is the same way we would halt selfDestuct.